### ISSUE TITLE
Unused js files

### ISSUE THREAD

**Issue by Tymvie** [ahmedkaludi/accelerated-mobile-pages#2001]
Hello,                                                                                                                                                                                          

When a post that contains the `[tweet tweet="638793490521001985"]` shortcode is splitted up into several pages using the `<!--nextpage-->` comment, all pages contain the `https://cdn.ampproject.org/v0/amp-twitter-0.1.js` file. This happens because the plugin passes the whole post content through the `the_content` filter.

This also creates problems for me because I develop a plugin that appends some staff using this filter and would like to see the part of the content that represents the current page and not the whole post content. Is it possible?

Thanks a lot

**Comment by ahmedkaludi** [comment]
@Tymvie where is this tweet shortcode coming from?

**Comment by Tymvie** [comment]
@ahmedkaludi this is coming from the AMP plugin from Automattic.
You could see the same behavior when using the `[youtube]` shortcode from  your plugin

**Comment by Tymvie** [comment]
Thanks a lot
Could we also support pages by replacing `is_single()` with `is_singular()`?

**Comment by MohammedKaludi** [comment]
@Tymvie yes and we are using it here https://github.com/ahmedkaludi/accelerated-mobile-pages/commit/9152e462684dfbb3f381154267146915759bcf87

**Comment by Tymvie** [comment]
@MohammedKaludi unfortunately, this does not solve the problem. We need to use` is_singular()` here: https://github.com/ahmedkaludi/accelerated-mobile-pages/commit/ab935b2f84014e4b5bad60d37b0499e6882f6e63#diff-6aa5512f7dd8d5ce60fc97b0f340912cR6464
